### PR TITLE
Fixes descriminator behaviour.

### PR DESCRIPTION
**Pull request by LeFnord**
- updates rubocop and makes it happy
- updates travis, align it with other gems

### CODE REVIEW

File: lib/grape-swagger/entity/helper.rb [hunk 1]
```diff
@@ -13,7 +15,7 @@ def model_name(entity_model, endpoint)
 
         def discriminator(entity_model)
           entity_model.superclass.root_exposures.detect do |value|
-            value.documentation.dig(:is_discriminator)
+            value.documentation&.dig(:is_discriminator)
```
Comment (LeFnord) [inline lib/grape-swagger/entity/helper.rb:18]: @Bugagazavr … this is the original change

**Comment by Bugagazavr** [comment]
Thank you, looks good, let me know when you ready to the new version of the gem to merge this branch. I also should ask about the minimum required version of the `grape-swagger`, it's a `1.4.0`, or maybe `1.2.0`?

**Comment by LeFnord** [comment]
yeah sorry, I mean 1.2.0, mixed it up with grape version

**Comment by Bugagazavr** [comment]
If you do not mind, I am ready to merge this branch and release `0.5.1` with the correct `grape-swagger` version (>= 1.2.0).

`0.5.0` will be yanked (do not worry nobody can install this because `grape-swagger` with version `1.4.0` doesn't exist)

**Comment by Bugagazavr** [comment]
Done https://rubygems.org/gems/grape-swagger-entity/versions/0.5.1 waiting for `grape-swagger` release.

**Pull request merged**